src/api/__tests__/Metadata.test.js

```diff
@@ -578,10 +579,7 @@ describe('api/Metadata', () => {
             metadata.getEditors = jest.fn().mockResolvedValueOnce('editors');
             metadata.getCustomPropertiesTemplate = jest.fn().mockReturnValueOnce('custom');
             metadata.getUserAddableTemplates = jest.fn().mockReturnValueOnce('templates');
-            metadata.getTemplates = jest
-                .fn()
-                .mockResolvedValueOnce('global')
-                .mockResolvedValueOnce('enterprise');
+            metadata.getTemplates = jest.fn().mockResolvedValueOnce('global').mockResolvedValueOnce('enterprise');
```

Comment:
Are those formatting changes necessary?

Reply:
Prettier changed them by default, not sure if I have it set up wrong or were the latest guidelines not applied to this file